README.md

```diff
@@ -1,2 +1,276 @@
+5	High Range                 (high byte)	(ignored if linear style = stepped)
+```
+
+### Set Linear Graph Threshold
+Configures a linear threshold. Up to 5 thresholds can be configured
+
+Note:
+* If the current value is greater than the last threshold, then the last threshold is selected.
+* The first threshold may have a threshold value >= 0; remaining thresholds must be > 0
+
+CAN ID: Base + 41
+
+```
+Offset  What                       Value
+======================================================================
+0	Threshold ID	           0 - 4
+1	Segment Length	           0 -> # of LEDs on device (ignored if linear style is ‘smooth’)
+2	Threshold                  (low byte)	
+3	Threshold                  (high byte)	
+4	Red	                   0 - 255
+5	Green	                   0 - 255
+6	Blue	                   0 - 255
+7	Flash Hz	           0 - 10 (0 = full on)
+```
+
+### Update Current Linear Graph Value
+Updates the current value for the linear graph
+
+CAN ID: Base + 42
+
+```
+Offset  What                       Value
+======================================================================
+0	Value                      (low byte)	
+1	Value                      (high byte)	
+```
+
+## Display functions
+
+### Set Display Value
+Set the current display value
+
+CAN ID: Base + 50
+
+```
+Offset  What                       Value
+======================================================================
+0	Digit Id    	           Must be 0
+0   Character                  ASCII value of character to display. 
+```
+
+The value of the display is set by the ASCII coded value. To set a numeric value, offset the value by 48: e.g. 48 = 0, 49 = 1, etc. 
+
+Most alphanumeric characters are supported as well, also via ASCII representation. 
+
+### Set Segment
+Discretely set segments on the display
+
+CAN ID: Base + 50
+
+```
+Offset  What                       Value
+======================================================================
+0	Digit Id    	           Must be 0
+1   Segment A                  0 to disable segment, 1 to enable
+2   Segment B                  0 to disable segment, 1 to enable
+3   Segment C                  0 to disable segment, 1 to enable
+4   Segment D                  0 to disable segment, 1 to enable
+5   Segment E                  0 to disable segment, 1 to enable
+6   Segment F                  0 to disable segment, 1 to enable
+7   Segment G                  0 to disable segment, 1 to enable
+```
+
+Sets individual segments for the display. Segments A-G conform to the standard 7 segment display segment identifications
+
+## Notifications
+Notifications related to events broadcasted from ShiftX3
+
+### Button State
+Indicates a change in the button state
+
+CAN ID: Base + 60
+
+```
+Offset  What                       Value
+======================================================================
+0	Button state	           1 = button is pressed; 0 = button
```

Comment:
offset should be 1 for Button ID, and should be "0 = button not pressed"

Reply:
Fixed. thanks for catching that!